### ISSUE TITLE
Discussion of JSON representation for objects returned by API

### ISSUE THREAD

**Issue by matthewhorridge** [who-icatx/icatx-project#126]
With reference to:   https://github.com/who-icatx/icatx-project/blob/main/docs/jsonForAPISpec.md

Could we make the key names for IRIs more uniform?  e.g. we have `entityURI` and `termId`. Could these just be replaced with `iri`?  (URI was superseded by IRI in 2012)

Cc @tudorache

**Comment by tudorache** [comment]
Yes, I know, we had a discussion with @cannn about this. This is the preference to use URI instead of IRI (they don't use the international part of the URIs). We use the ``termId``, because we return terms, and it is part of the domain vocabulary, and known by the users.
